.gitmessage

```diff
@@ -0,0 +1,23 @@
+###
```

Comment:
Let's not work with .gitmessage, as that will only work in certain cases and needs manual setup. If we want to address the commit messages, let's add an issue to get a proper solution in like http://commitizen.github.io/cz-cli/